### PR TITLE
[ODS-4268] Release tasks for ODS/API 5.0.0

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1 [hunk 1]
```diff
@@ -8,8 +8,8 @@
 $ErrorActionPreference = 'Stop'
 
 $toolVersion = @{
-    dbDeploy = "2.0.0-b10031"
-    codeGen = "5.0.0-b10352"
+    dbDeploy = "2.0.0-b10033"
```
Comment (tbmeador-dlp) [inline Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1:11]: needs to be `2.0.0`

everything else should be `5.0.0`

Comment (semalaiappan) [reply Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1:11]: OK

Comment (semalaiappan) [reply Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1]: done . Thanks

Comment (jamessmckay) [reply Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1]: agreed

**Pull request merged**